library/src/fy_library/mixins/property/parsed_fy_py_files_with_own_abstract_mixin/using_parsed_fy_py_files_map_by_key_fy.py

```diff
@@ -100,6 +100,15 @@ def with_own_abstract_mixin_if_exists(
                         ],
                     )
 
+                    for (
+                        abstract_mixin
+                    ) in parsed_property_fy_py_file.abstract_property_mixins:
+                        if (
+                            abstract_mixin.python_class_name.pascal_case
+                            == parsed_abstract_property_fy_py_file.python_class_name.pascal_case
+                        ):
+                            return parsed_fy_py_file
```

Comment:
Convert to ist comprehension with `any` operator.
```suggestion
any([abstract_mixin.python_class_name.pascal_case
                            == parsed_abstract_property_fy_py_file.python_class_name.pascal_case
                    for (
                        abstract_mixin
                    ) in parsed_property_fy_py_file.abstract_property_mixins
])
```

Reply:
Changed